src/components/Badge/styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+import { theme } from '../../theme';
+
+export const RoundedBadge = styled.div`
```

Comment:
It should be an inline block instead of a block, so either:
- make it a `styled.span`
- add `display: inline-block` to it

Reply:
It’s a draft :) I’ve added the wip tag. No need to review this PR for now, thanks !

Reply:
My bad 🙊